src/datapump/jobs/geotrellis.py

```diff
@@ -86,7 +86,13 @@ class GeotrellisJob(Job):
     result_tables: List[AnalysisResultTable] = []
 
     def next_step(self):
-        if self.step == JobStep.starting:
+        if self.start_time + self.timeout < datetime.now():
```

Comment:
Hmm, shouldn't this comparison be the other way around? So ">"?

Reply:
I had the same confusion, maybe I'm writing this in a weird way... but if the start time was 3 hours ago, and the timeout is two hours ago, start time + timeout is 1 hour ago. This is less than now, so you've gone past timeout. Does that make sense?

Reply:
Oh, I see. Gotcha.